src/utils/trendingMoviesSchema.js

```diff
@@ -0,0 +1,6 @@
+import { normalize, schema } from 'normalizr'
+
+const movieSchema = new schema.Entity('movies')
+const moviesListSchema = new schema.Array(movieSchema)
+
+export const trendingMoviesSchema = data => normalize(data, moviesListSchema)
```

Comment:
i am not sure `normalize` call should be here. this code is related to the operation